ocaml/parsing/jane_syntax_parsing.mli

```diff
@@ -105,11 +105,26 @@ end
     also why we don't expose any functions for rendering or parsing these names;
     that's all handled internally. *)
 module Embedded_name : sig
-  (** A nonempty list of name components, without the leading root component
-      that identifies it as part of the modular syntax mechanism.  This is a
-      nonempty list corresponding to the different components of the name: first
-      the feature, and then any subparts. *)
-  type t = ( :: ) of string * string list
+  module Erasability : sig
+    type t =
+      | Erasable
+      | Non_erasable
+  end
```

Comment:
The documentation for this module should live in the `.mli` file, not the `.ml` file